tsconfig.json

```diff
@@ -1,32 +1,34 @@
 {
-	"version": "1.5.0",
-	"compilerOptions": {
-		"declaration": false,
-		"module": "umd",
-		"noImplicitAny": true,
-		"outDir": "_build/",
-		"removeComments": false,
-		"sourceMap": true,
-		"target": "es5"
-	},
-	"filesGlob": [
-		"./src/**/*.ts",
-		"./tests/**/*.ts",
-		"./typings/tsd.d.ts"
-	],
-	"files": [
-		"./src/main.ts",
-		"./tests/functional/all.ts",
-		"./tests/intern-local.ts",
-		"./tests/intern.ts",
-		"./tests/typings/chai/chai.d.ts",
-		"./tests/typings/digdug/digdug.d.ts",
-		"./tests/typings/dojo2/dojo.d.ts",
-		"./tests/typings/intern/intern.d.ts",
-		"./tests/typings/leadfoot/leadfoot.d.ts",
-		"./tests/typings/node/node.d.ts",
-		"./tests/typings/tsd.d.ts",
-		"./tests/unit/all.ts",
-		"./typings/tsd.d.ts"
-	]
+    "version": "1.5.0",
```

Comment:
If Atom and Atom-typescript were being used for these changes, that would probably be the reason why.  It always outputs tsconfig.json with soft tabs, and it rewrites it any time the file is changed.

Grunt should now be taking care of generating this when we run `grunt dev`, but Atom-typescript will immediately overwrite that (since it does so whenever the file changes).  I'm not aware of a way to switch that off other than closing Atom entirely (or disabling Atom-typescript).